server/database/data/build.sql

```diff
@@ -0,0 +1,32 @@
+BEGIN;
+
+DROP TABLE IF EXISTS USERS CASCADE;
+DROP TABLE IF EXISTS PRODUCTS CASCADE;
+DROP TABLE IF EXISTS CART CASCADE;
+
+CREATE TABLE IF NOT EXISTS USERS(
+    USER_ID SERIAL PRIMARY KEY,
+    USERNAME VARCHAR(28) UNIQUE NOT NULL,
+    F_NAME TEXT NOT NULL,
+    EMAIL TEXT UNIQUE NOT NULL,
```

Comment:
It's good to be VARCHAR type  because I think ```text``` type consumes more memory (heap)

Reply:
A good note to take in consideration :)